CONTRIBUTING.md

```diff
@@ -24,14 +28,17 @@ convert -resize 256 my_big_image.jpg my_image.jpg
 ffmpeg -i my_big_image.jpg -vf scale=w=256:h=256:force_original_aspect_ratio=decrease my_image.png
 ```
 
-Then just run the `./autogen.sh` to generate the added or updated node classes.
+Then just run the `./autogen.sh` to generate the added or updated node classes. (cf. [DEVELOPMENT.md][DEVELOPMENT.md])
```

Comment:
+1. We don't have to add `.md` suffix to link title

Reply:
:ok_hand:

Reply:
done!